### ISSUE TITLE
Windows meson build fails due to recent commit

### ISSUE THREAD

**Issue by paxcut** [radareorg/radare2#16372]
# This template is meant for bug reports, if you have a feature request, please be as descriptive as possible and delete the template

Make sure you are testing using the latest git version of radare2 before submitting any issue.

*If you would like to report a bug, please fill the template below*

### Work environment

| Questions                                            | Answers
|------------------------------------------------------|--------------------
| OS/arch/bits:                                                      windows x86_64 64
| File format of the file you reverse                          PE
| Architecture/bits of the file                                 x86/64
| r2 -v full output, **not truncated** (mandatory)     | radare2 4.4.0-git 24070 @ windows-x86-64 git.4.3.1-157-gcd2715dc9
commit: cd2715dc953257261113f904eab93695fa778d26 build: Mon 03/30/2020__13:26:18.03

### Expected behavior
The following command should output a working r2 executable:

msbuild build\radare2.sln /p:Configuration=Debug /m

### Actual behavior
command given above gives as output:

     "h:\temp\radare2\build\radare2.sln" (default target) (1) ->
       "h:\temp\radare2\build\binr\r2r\3782362@@r2r@exe.vcxproj" (default target) (305) ->
       (Link target) ->
         run.c.obj : error LNK2019: unresolved external symbol r2r_subprocess_stdin_write referenced in function r2r_ch
       eck_json_test [h:\temp\radare2\build\binr\r2r\3782362@@r2r@exe.vcxproj]
         .\r2r.exe : fatal error LNK1120: 1 unresolved externals [h:\temp\radare2\build\binr\r2r\3782362@@r2r@exe.vcxpr
       oj]

    371 Warning(s)
    2 Error(s)

### Steps to reproduce the behavior 
setup the build as explained in the radare2 book and attempt to build executables. 

The unresolved external occurs in the r2r library  because when building for windows there is no stdin defined, but r2r_check_jason_test calls r2r_subprocess_stdin_write which is not defined (part of #else from #if Windows). Removing this call produces a working executable sans jason r2r testing. I haven't been able to figure out what should be written instead of the offending stdin call to fix this completely.

**Comment by radare** [comment]
Should be fixed in my PR. but if you know how to fix a bug dont open an issue, just do a PR next time

**Comment by paxcut** [comment]
. All I did was get rid of the function so that the compilation could finish, but I tried to figure how to do the windows equivalent of the stdin operation, but I just started looking a radare two days ago and there was nothing similar anywhere I looked. I suspected that  an expert could fix it in an instant. I realize that in a sense this wasn't an issue, but I still wanted to help if only to make the non-issue stand out more. I am sorry if I gave the wrong impression.
